marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.tsx

```diff
@@ -251,6 +251,22 @@ const EntityMapTable: React.FC<Props> = props => {
                     XPath Expressions
                   </a>
                 </li>
+                <li>
```

Comment:
From the ticket, shouldn't line 243 to 253 be deleted? They idea is to replace that li wtih this new li, correct?

Reply:
ikr, we should have images in the preview of the tickets...

![image](https://user-images.githubusercontent.com/108084664/227593499-e881e9f7-f0ce-4341-9f89-69dbb47ad3c8.png)

this is what i had to do, and later christian ask for changing "extraction" to "custom".

Reply:
The code looks correct! Title was misleading but the idea is to add another link to display documentation for custom functions. Basically your comment with the screenshot but changing "extraction" to "custom" is correct @littlecherro.